ceopardy/api.py

```diff
@@ -0,0 +1,97 @@
+import flask
+from flask import Blueprint, request
+from flask_socketio import SocketIO, send, emit
+import random
+#import flask_login
+
+
+api = Blueprint('api', __name__, url_prefix='/api')
+
+@api.route('/', defaults={'path': 'debug'})
+@api.route('/<path:path>', methods=['GET', 'PUT', 'POST', 'DELETE'])
+#@flask_login.login_required
+def dispatcher(path):
```

Comment:
This is a lot of low-level code that seems to do very little if not nothing. I'm assuming these are tests and will not rigorously review it.